server.py

```diff
@@ -0,0 +1,266 @@
+from collections import defaultdict
+import datetime
+from eyeball_pp.classes import TASK_OUTPUT_KEY, FeedbackResult
+from eyeball_pp.eval import SUCCESS_CUTOFF
+from eyeball_pp.recorders import Checkpoint, EvalRecorder, FileRecorder
+from eyeball_pp.system_state import bucketize_checkpoints
+from eyeball_pp.utils import time_to_str
+import json
+import pandas as pd
+from statistics import variance
+import streamlit as st
+import sys
+from typing import Optional, Any
+
+
+@st.cache_data
+def get_recorder() -> EvalRecorder:
```

Comment:
you can change this to eyeball_pp.default_recorder after you rebase from main, i just pushed a change.